subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/CrossVersionPerformanceTestRunner.groovy

```diff
@@ -95,6 +97,16 @@ class CrossVersionPerformanceTestRunner extends PerformanceTestSpec {
         this.releases = releases
         this.buildContext = buildContext
         this.testProject = TestScenarioSelector.loadConfiguredTestProject()
+
+        // Pause between builds for the following reasons:
+        //
+        // - better simulate a normal development lifecycle where builds are rarely executed
+        //   back-to-back, as there is typically some work being done between the builds
+        // - give file system events a chance to arrive; otherwise we might detect them during
+        //   the build, and invalidating the VFS
+        // - let VFS cleanup (that happens between builds) finish
+        //
+        addBuildMutator { __ -> new DelayBeforeBuildMutator(500, MILLISECONDS) }
```

Comment:
🤔  Should we set that mutator directly in the `GradleBuildExperimentRunner`?
Otherwise other runners like `CrossBuildPerformanceTestRunner` doesnt' have this delay I believe

Reply:
Good point, I updated the PR, PTAL.